src/constants.js

```diff
@@ -18,3 +18,19 @@ export const KEG_NAME_BY_KEG_TYPE: { [KegType]: string } = {
   SixthBarrel: 'Sixth Barrel Keg',
   SlimQuarter: 'Slim Quarter Keg',
 };
+
+export const DESCRIPTION_BY_DEVICE_STATE: { [key: DeviceStatus]: string } = {
+  Active:
+    'The Brewskey Box is currently in the standard mode.' +
+    ' If you have a valve it will be closed and the sensors will' +
```

Comment:
really minor detail but I'd rather have the spaces at the end of the sentence instead of the beginning.